components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -103,7 +103,14 @@ export const PrebuildDetailPage: FC = () => {
     }, [isTriggerError, triggerError, toast]);
 
     const prebuildPhase = useMemo(() => {
-        const name = currentPrebuild?.status?.phase?.name;
+        if (!currentPrebuild) {
+            return {
+                icon: <LoadingState size={20} delay={false} />,
```

Comment:
We have another `unknown` in this page: unknown repository name

For that places (places where they will present a string), will [`animate-pulse`](https://tailwindcss.com/docs/animation#pulse) be better?

Reply:
Or make it simple, if it's loading, then show a empty string

Reply:
Made the breadcrumb empty if we do not have the info yet, makes sense. Not sure about introducing skeletons since we do not usually do it.